homeassistant/components/swiss_public_transport/config_flow.py

```diff
@@ -24,6 +41,18 @@
     }
 )
 
+OPTIONS_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_RATE, default=DEFAULT_RATE): NumberSelector(
+            NumberSelectorConfig(min=0, mode=NumberSelectorMode.BOX)
+        )
+    },
+)
```

Comment:
Adding custom polling rates in the UI (also known as, scan interval) is no longer allowed.

If you wish to use a different interval, you can define your own, based on your own rules and schedule, using an automation that call the home assistant update entity service.